pact/Pact/Core/SizeOf.hs

```diff
@@ -381,7 +382,8 @@ instance SizeOf Yield
 instance SizeOf DefPactExec
 
 -- spaninfo
-instance SizeOf SpanInfo
+instance SizeOf SpanInfo where
+  sizeOf _ _ _ = pure 0
```

Comment:
Charging 0 for a structure that accounts for something like ~30% of the actual module size on disk seems wrong to me.